benchmarks/CalendarBench.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+use Punic\Calendar;
+
+class CalendarBench
+{
+    /**
+     * @Revs(100)
+     * @Iterations(5)
+     */
+    public function benchEraName()
```

Comment:
You can move these annotations to the class block:

```php
/**
 * @Revs(10000)
 * @Iterations(10)
 */
class ...
```

Also I would use more revolutions and iterations as indicated above. You might also consider creating an abstract class with these annotations if all your benchmarks have the same requirements.